core/modules/wiki-bulkops.js

```diff
@@ -13,10 +13,38 @@ Bulk tiddler operations such as rename.
 "use strict";
 
 /*
+Retarget tags and lists from fromTitle toTitle.
+*/
+exports.retargetReferences = function(fromTitle,toTitle) {
```

Comment:
hi @edrex, Thanks for this. Looks very useful. Any demo somewhere?

May I ask, if you are providing a hook already, why not use the hook mechanism to actually handle the renaming? ...this would also be a great example for an "implementing a hook" roundtrip.